challenge_14/java/jdfurlan/src/ReversingALinkedList.java

```diff
@@ -0,0 +1,71 @@
+
+import java.util.Scanner;
+
+public class ReversingALinkedList {
+
+	private static void printList(ListNode head) {
+		while (head != null) {
+			System.out.print(head.data);
```

Comment:
Generally separators aren't included in the parsed data. As you say, it's not too big of a deal for this challenge.